src/Consumer/CreateHomedirConsumer.php

```diff
@@ -54,6 +64,133 @@ public function __construct(
     * @return boolean True if success, false otherwise.
     */
     public function execute(AMQPMessage $message)
+    {
+        if ('NFS' === $this->filesystemType) {
+            return($this->processNFS($message));
+        } else {
```

Comment:
Not a huge deal, but the other setting could be anything else, but is expected to be "Linux" as set in the .env file.